### PR TITLE
Release 1.7 verification 105

### PR DESCRIPTION
**Pull request by chinhuang007**
This PR takes the latest change from master to rel-1.7.0 branch along with updated version number and preview domain name. The version number will be changed back to 1.7.0 after release candidate passes verification.

### CODE REVIEW

**Comment by codemzs** [comment]
@chinhuang007 It seems to only have last commit from master? I don’t see SoftmaxCrossEntopyLoss for example.

**Comment by chinhuang007** [comment]
@codemzs Please be specific on what is missing. My branch is 1 commit ahead of master. So it is a superset of master.

**Comment by codemzs** [comment]
@chinhuang007 That clarifies. Thanks.

File: VERSION_NUMBER [hunk 1]
```diff
@@ -1 +1 @@
-1.7.0
+1.7.105
```
Comment (wschin) [inline VERSION_NUMBER:1]: Is this change intended? Does 105 have a special meaning?

Comment (chinhuang007) [reply VERSION_NUMBER:1]: Yes, this is for the version number in test pypi. We have to use a different number each time we have a release candidate. Once the final verification is complete, we will change it back to 1.7.0 in the release branch.

Comment (chinhuang007) [reply VERSION_NUMBER:1]: There is no special meaning other than I started from 101 and increment by 1 for each release candidate.
